pkg/client/mode.go

```diff
@@ -31,17 +31,25 @@ type Mode string
 
 const (
 	// Quick runs a single E2E test and the systemd log tests.
-	Quick Mode = "Quick"
+	Quick Mode = "quick"
 
-	// Conformance runs all of the E2E tests and the systemd log tests.
-	Conformance Mode = "Conformance"
+	// NonDisruptiveConformance runs all of the `Conformance` E2E tests which are not marked as disuprtive and the systemd log tests.
+	NonDisruptiveConformance Mode = "non-disruptive-conformance"
+
+	// CertifiedConformance runs all of the `Conformance` E2E tests and the systemd log tests.
+	CertifiedConformance Mode = "certified-conformance"
 )
 
-const defaultSkipList = `Alpha|\[(Disruptive|Feature:[^\]]+|Flaky)\]`
+// nonDisruptiveSkipList should generally just need to skip disruptive tests since upstream
+// will disallow the other types of tests from being tagged as Conformance. However, in v1.16
+// two disruptive tests were  not marked as such, meaning we needed to specify them here to ensure
+// user workload safety.
```

Comment:
A link to the relevant issues/PRs/discussions here would be useful for coming back to this in the future.

Reply:
👍 Added.